### ISSUE TITLE
Unable to use docker 

### ISSUE THREAD

**Issue by NablaMetal** [nanos/FediFetcher#152]
Hi,
I followed carefully the readme, and got the python script working, but I don't manage to get the container to fetch any toots.
I tried docker run, or the docker-compose file, filled out the token and server, but the script says it's finishes almost instantly :
```
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:57 UTC: Starting FediFetcher
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:57 UTC: Processing finished in 0:00:00.003125.
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:58 UTC: Starting FediFetcher
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:58 UTC: Processing finished in 0:00:00.001891.
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:59 UTC: Starting FediFetcher
fedifetcher-fedifetcher-1  | 2024-07-29 15:12:59 UTC: Processing finished in 0:00:00.001449.
```
I's like to use docker because I like to keep my host clean as much of possible without any extra package.
If I understand correctly, if I use the container, I don't need to clone the git repo first right ?
Basically it should work only with  `docker run -it ghcr.io/nanos/fedifetcher:latest --access-token=myaccesstokenstring --server=mastodon.myserver.org` ?
Thank you

**Comment by nanos** [comment]
You will need to add at least one option, such as `--home-timeline-length=200`. By default everything is 'off', so you'd expect it not do anything.

The [recommended options](https://github.com/nanos/FediFetcher?tab=readme-ov-file#configuration-options), translated for the command line would, be

```bash
docker run -it ghcr.io/nanos/fedifetcher:latest --home-timeline-length=200 --max-followings=80 --from-notifications=1 --access-token=myaccesstokenstring --server=mastodon.myserver.org
```

**Comment by NablaMetal** [comment]
Of course it was something obvious ;) Thanks !!

**Comment by paulo-roger** [comment]
> ```shell
> --home-timeline-length=200 --max-followings=80 --from-notifications=1
> ```

Could I supply this values in a .env?

**Comment by nanos** [comment]
I’m not really familiar with docker, so I can only offer limited support in this myself. 

However there are a couple of examples on running FediFetcher using [docker compose](https://github.com/nanos/FediFetcher/blob/main/examples/docker-compose.yaml) and [k8](https://github.com/nanos/FediFetcher/blob/main/examples/k8s-cronjob.yaml)

does any of this help?

**Comment by nanos** [comment]
@paulo-roger Since v 7.1.6 you can configure FediFetcher through environment variables:

https://github.com/nanos/FediFetcher/releases/tag/v7.1.6

Maybe this helps?
